Source/VFECore/Abilities/Ability.cs

```diff
@@ -123,7 +123,9 @@ public virtual float GetPowerForPawn()
         {
             float power = CalculateModifiedStatForPawn(this.def.power, this.def.powerStatFactors, this.def.powerStatOffsets);
             var multiplier = this.def.GetModExtension<AbilityExtension_RandomPowerMultiplier>();
-            return multiplier == null ? power : Rand.Range(power * multiplier.min, power * multiplier.max);
+            if (multiplier == null) return power;
+            FloatRange range = multiplier.range;
+            return power * Rand.Range(range.min, range.max);
```

Comment:
```suggestion
            return power * multiplier.range.RandomInRange;
```
There's a provided function for this. :)

Reply:
Much cleaner, thank you!